apps/golden-sample-app/src/app/transfer/transfer-journey-bundle.module.ts

```diff
@@ -14,6 +15,7 @@ import { JourneyCommunicationService } from '../services/journey-communication.s
       useValue: {
         maskIndicator: false,
         maxTransactionAmount: 100,
+        slimMode: !environment.common.designSlimMode,
```

Comment:
why do we negate it? it's a bit more straightforward if the semantics of the parameter are the same in environment.ts and the journey config...

Reply:
yes, indeed. I realized the logic was opposite, so I fixed it. Can you give another look?